projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -569,10 +567,6 @@
 	margin: 0;
 }
 
-.jetpack-option__type.jetpack-option__type {
```

Comment:
This selector was duplicated above. 

Even though the above selector is nested in `.jetpack-field`, I don't see any control where `.jetpack-option__type ` is outside a field.